contracts/RooibosDao.sol

```diff
@@ -31,13 +31,17 @@ contract RooibosDao is Initializable, GovernorUpgradeable,
         __Governor_init("RooibosDao");
         __GovernorVotes_init(_token);
         __GovernorCountingSimple_init();
-        __GovernorVotes_init(_token);
+        __GovernorVotes_init(_token); // Repeat?
         __GovernorVotesQuorumFraction_init(4); // 4% required for quorum
     }
 
     // Function for creating proposal
     // Function for adjusting quorum fraction
 
+    function adjustQuorum(uint256 quorumIn) internal {
```

Comment:
I think there might actually be a `setQuorumFraction` in the `GovernorVotesQuorumFraction` interface. Worth checking so we don't end up duplicating it.